Source/MLXNN/Transformer.swift

```diff
@@ -158,9 +158,9 @@ class TransformerEncoderLayer: Module {
         } else {
             y = attention(x, keys: x, values: x, mask: mask)
             y = dropout1(y)
-            y = ln1(x + y)
+            x = ln1(x + y)
 
-            y = linear1(y)
+            y = linear1(x)
```

Comment:
Wow I'm impressed you caught all these changes.

Reply:
Going through the diffs works pretty well:

- https://github.com/ml-explore/mlx/compare/v0.16.0...v0.17.3

Some may still sneak through, but I _think_ it is pretty close to in sync.